gildniy/WayFarer
# #167122102 Build Auth (Login/Register) UI page

#### What does this PR do?
Having the Authentication (Login / Register) UI pages
#### Description of Task to be completed?
The following are the available features:

On Login form:
- Username (string type) input field
- Password (password type) input field
- Remember me (checkbox type) input field
- Enter (submit type) input field
- A link to the Register form
- Basic form validation

On Register form
- Username (string type) input field
- Email (email type) input field
- Password (password type) input field
- Password confirmation (password type) input field
- Register (submit type) input field
- A link to the Login form
- Basic form validation
#### How should this be manually tested?
After cloning the repo, you can open the UI directory and open the `index.html` page in the browser
- You can type in the text and password type field, and check the checkbox. Also, submit the form either filled or empty to observe notifications and validation errors.
#### Any background context you want to provide?
For any application requiring the user activities and to know or assign the activity to the specific user, an authentication system is needed.
#### What are the relevant pivotal tracker stories?
#167122102
#### Screenshots

![Login](https://user-images.githubusercontent.com/3137192/60787570-8743b200-a15a-11e9-8255-4187cbdaddd6.PNG)

![image](https://user-images.githubusercontent.com/3137192/60787621-b823e700-a15a-11e9-8927-8dc7b1598472.png)
